### DISCUSSION TITLE
Remove Logo (Avatar) doesn't work

### DISCUSSION THREAD

**Discussion by addvanced** [outline/outline#7159] [Self Hosting]
When I try to remove the logo (avatar) of my workspace, it fails.

I get a BadRequestError: avatarUrl: Expected string, received null

So, it seems like removing the logo sets the avatarUrl to null, but it expects a string. So a validation issue.

I am running v0.77.2, and storage is a S3 Bucket (Space) at DigitalOcean.

![outline_error](https://github.com/outline/outline/assets/37810041/150090a1-2ed0-4d36-8038-fda440db728a)

**Comment by tommoor** [comment]
Reported and fixed here for next release https://github.com/outline/outline/pull/7124
